exercises/variable-length-quantity/variable_length_quantity.py

```diff
@@ -1,6 +1,6 @@
-def encode():
+def encode(numbers):
     pass
 
 
-def decode():
+def decode(bytes_):
```

Comment:
Why not just use `bytes`?
Could you please explain your reasoning?

Reply:
In python 3.6, `bytes()` is a built-in function https://docs.python.org/3/library/functions.html#func-bytes, and I didn't want to shadow that built-in function.

In line with [PEP8](https://www.python.org/dev/peps/pep-0008/#function-and-method-arguments) I added a trailing underscore:

> If a function argument's name clashes with a reserved keyword, it is generally better to append a single trailing underscore rather than use an abbreviation or spelling corruption. Thus class_ is better than clss. (Perhaps better is to avoid such clashes by using a synonym.)

Reply:
@ackerleytng now I understand, thanks!